src/app/core/record-works/record-works.service.ts

```diff
@@ -105,63 +105,49 @@ export class RecordWorksService {
     this.sortAsc = options.sortAsc
 
     this._$loading.next(true)
-    this._togglz
```

Comment:
Since this is the only use of `_togglz` we can probably remove the dependency injection of this